docs/sources/operators-guide/deploy-grafana-mimir/getting-started-helm-charts/_index.md

```diff
@@ -79,6 +79,14 @@ Using a custom namespace solves problems later on because you do not have to ove
          # empty, disabled.
    ```
 
+   > **Note:** To see all of the configurable parameters for a Helm chart installation, use `helm show values grafana/mimir-distributed`.
+   > If you want to get the values file from Github, build the URL as follows because we git tag every release:
+   > `https://github.com/grafana/mimir/blob/mimir-distributed-<chart-version>/operations/helm/charts/mimir-distributed/values.yaml`.
+   > For example, `https://github.com/grafana/mimir/blob/mimir-distributed-3.1.0/operations/helm/charts/mimir-distributed/values.yaml`.
+   > You must use a matching Helm chart version, because charts are not guaranteed to be backwards compatible between major version.
```

Comment:
IIRC we don't really guarantee any backwards compatibility of values files with older versions of the chart. Not only for major versions, but for all. I think the biggest potential breaking change is referencing a new named template in e.g. the nginx config (`nginx.nginxConfig.file`). If you use that new config with any old chart, then it will break. So I think we should change this sentence.

```suggestion
   > You must use a matching Helm chart version, because chart values are not guaranteed to be backwards compatible between versions.
```

Is this your understanding too?

Reply:
Yes, that's true. The thing that breaks here is when a new named template is used on the old chart.